index.js

```diff
@@ -394,7 +391,26 @@ class ClientServiceBase {
           startAge,
           endAge,
       );
-      return this._validateLedger(request);
```

Comment:
This function sends the validation requests to Ledger and Auditor and then compares the response. We don't need it anymore since we always do the linearizable validation in the Auditor mode.